src/tab/Tab.tsx

```diff
@@ -16,20 +16,26 @@ export interface TabProps {
   items: TabItem[];
   children: React.ReactNode[];
   testid?: string;
-  initialActiveTabIndex?: number;
+  selectedItem?: number;
   customClassName?: string;
+  onTabChange?: (index: number) => void;
 }
 
 function Tab({
   testid,
   items,
-  initialActiveTabIndex = 0,
+  selectedItem = 0,
   children,
-  customClassName
+  customClassName,
+  onTabChange
 }: TabProps) {
-  const [activeTabIndex, setActiveTabIndex] = useState(initialActiveTabIndex);
+  const [activeTabIndex, setActiveTabIndex] = useState(selectedItem);
   const tabClassName = classNames("tab", customClassName);
 
+  useEffect(() => {
+    setActiveTabIndex(selectedItem);
+  }, [selectedItem]);
```

Comment:
I think this is OK for now, but I don't think a component being both controlled and uncontrolled is a very good idea. Let's discuss this in the next team meeting and finalize the decision.